garak/resources/beast/beast_attack.py

```diff
@@ -342,7 +347,7 @@ def run_beast(
     suffix_len: int = 40,
     data_size: int = 20,
     target: Optional[str] = "",
-    outfile: str = beast_resource_data / "suffixes.txt",
+    outfile: Path = beast_resource_data / "suffixes.txt",
```

Comment:
if this is (expensive) results from a BEAST run, should it go in xdg `data`?

Reply:
Not sold on that as this data is being `built` on the system for the run and could be extracted from the `.report.jsonl` prompts of a run that used them.  There is currently no check for if the file exists prior to building it, and the data is only valid for the specific model it was used with.  I do think there could be some future enhancement that makes this more `data` class than `cache` just not based the current implementation and scope of this PR.

A clear output value that documents to the user where this was placed may be appropriate in a future expansion.